cassini_v2_files/mission_phase_name.py

```
@@ -0,0 +1,268 @@
+    replacements = {
+        r"PDS4_PDS_1.*?\.sch":      "PDS4_PDS_1O00.sch",
+        r"PDS4_DISP_1.*?\.sch":     "PDS4_DISP_1O00_1510.sch",
+        r"PDS4_SP_1.*?\.sch":       "PDS4_SP_1O00_1320.sch",
+        r"PDS4_CASSINI_1.*?\.sch":  "PDS4_CASSINI_1O00_1800.sch",
+        r"PDS4_PDS_1.*?\.xsd":      "PDS4_PDS_1O00.xsd",
+        r"PDS4_DISP_1.*?\.xsd":     "PDS4_DISP_1O00_1510.xsd",
+        r"PDS4_SP_1.*?\.xsd":       "PDS4_SP_1O00_1320.xsd",
+        r"PDS4_CASSINI_1.*?\.xsd":  "PDS4_CASSINI_1O00_1800.xsd"
+    }
+
+    with open(file_path, "r", encoding="utf-8") as f:
```

Ruff (0.15.6)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)